multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/v2/ServicesCloudModelBuilder.java

```diff
@@ -143,5 +143,36 @@ protected String getInvalidServiceConfigTypeErrorMessage(String serviceName, Obj
                                     Map.class.getSimpleName(), serviceParameters.getClass()
                                                                                 .getSimpleName());
     }
+    
+    protected static class CommonServiceProperties {
```

Comment:
Shouldn't this be called `CommonResourceParameters`?

Reply:
Done